core/frontend/render-mode-manager.php

```diff
@@ -75,12 +75,23 @@ public function get_current() {
 		return $this->current;
 	}
 
+	/**
+	 * @param Render_Mode_Base $render_mode
+	 *
+	 * @return $this
+	 */
+	public function set_current( Render_Mode_Base $render_mode ) {
```

Comment:
```suggestion
	private function set_current( Render_Mode_Base $render_mode ) {
```

Reply:
This one creates a quick way to define the render mode without any $_GET params.